tests/run-all-tests.sh

```diff
@@ -12,4 +12,6 @@ bash all-up-test.sh VALIDATOR_OUT
 bash all-up-test.sh LONDON
 # An alchemy api key is necessary for this arbitrary logic test, so we are disabling it
 #bash all-up-test.sh ARBITRARY_LOGIC
+# An alchemy api key is necessary for this relay market test, so we are disabling it
+#bash all-up-test.sh RELAY_MARKET
```

Comment:
this should probably have 'RELAY_MARKET API_KEY' to make it clear how it's intended to be called. Or we could add a bash conditional to only run it when the value is set, probably not needed though.